src/compiler/types.ts

```diff
@@ -929,18 +1068,36 @@ namespace ts {
         _statementBrand: any;
     }
 
+    // @kind(SyntaxKind.EmptyStatement)
+    export interface EmptyStatement extends Statement { }
+
+    // @kind(SyntaxKind.DebuggerStatement)
+    export interface DebuggerStatement extends Statement { }

```

Comment:
I'd rather not litter all of the interfaces with brands. They seem more valuable when disambiguating structurally similar nodes that may not be legally valid in certain parts of the tree (such as `LeftHandSideExpression` vs `Expression`).

If we ever end up with Enum type literals, I might change it to:
```ts
export interface EmptyStatement extends Statement {
    kind: SyntaxKind.EmptyStatement;
}
...
```